<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUp_recruit</class>
 <widget class="QDialog" name="SignUp_recruit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>330</width>
     <height>347</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_name">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_title">
        <property name="text">
         <string>Title</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_company">
        <property name="text">
         <string>Company</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_url">
        <property name="text">
         <string>Url</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_description">
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_name"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_title"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_company"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_url"/>
      </item>
      <item row="4" column="1">
       <widget class="QTextEdit" name="textEdit_details"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_finish">
      <property name="text">
       <string>Finish</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
